src/lib/resolver.js

```diff
@@ -15,53 +15,21 @@ var PDI = '\u2069';
 const meta = new WeakMap();
 const resolutionChain = new WeakSet();
 
-export function createEntry(node, lang) {
-  var keys = Object.keys(node);
+export function createEntry(id, entity, lang) {
+  meta.set(entity, {id, lang});
```

Comment:
Ah, no, sorry, my bad.  We don't have `id` on entity anymore ^^.  Ignore the previous comment.